hbase-server/src/test/java/org/apache/hadoop/hbase/client/hamcrest/BytesMatchers.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.hadoop.hbase.client.hamcrest;
+
+import static org.hamcrest.core.Is.is;
```

Comment:
This only reason for import?

Reply:
I don't follow the question. This class comes out of hamcrest-core.

Reply:
Is this the only class we use from hamcrest? Are we adding a new dependency to modules just for one class? Its a good one I presume.

Reply:
_is_ is provided by the existing dependency, `hamcrest-core`. I've added the dependency on `hamcrest-library` for the the collection- and bean-related matchers that it provides. This patch makes use of _contains_ and _hasProperty_, but I look forward to writing more tests that make use of others. It's quite unusual to depend only on `hamcrest-core`; usually `-core` and `-library` come as a pair in user applications. I believe `-core` is stand-alone so that 3rd party extension libraries (such as Spotify's [java-hamcrest](https://github.com/spotify/java-hamcrest) can pull in the minimal set if they so choose.

Reply:
ok